module_5/oop.py

```diff
@@ -0,0 +1,5 @@
+from oop_classes import PdqeNewsPaperSite
+
+# Note! please insert into parameter desired file location
+module_5 = PdqeNewsPaperSite("/Users/shaliyu1/Documents/python_for_dqe/module_5/newspaper.txt")
```

Comment:
This is a good note.
But if I don't specify the path, then my code won't work.
It is better that the file is created automatically in the directory.  But optionally, you can set a directory to choose from.

Reply:
fixed